src/apps/experimental/features/details/components/buttons/SplitVersionsButton.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { FC, useCallback } from 'react';
+import { IconButton } from '@mui/material';
+import CallSplitIcon from '@mui/icons-material/CallSplit';
+import { useQueryClient } from '@tanstack/react-query';
+import { useDeleteAlternateSources } from 'hooks/api/videosHooks';
+import globalize from 'lib/globalize';
+import confirm from 'components/confirm/confirm';
+import loading from 'components/loading/loading';
+
+interface SplitVersionsButtonProps {
+    paramId: string;
+    queryKey?: string[];
+}
+
+const SplitVersionsButton: FC<SplitVersionsButtonProps> = ({
+    paramId,
+    queryKey
+}) => {
+    const queryClient = useQueryClient();
+    const deleteAlternateSources = useDeleteAlternateSources();
+
+    const splitVersions = useCallback(() => {
+        confirm({
+            title: globalize.translate('HeaderSplitMediaApart'),
+            text: globalize.translate('MessageConfirmSplitMediaSources')
+        })
+            .then(function () {
+                loading.show();
+                deleteAlternateSources.mutate(
+                    {
+                        itemId: paramId
+                    },
+                    {
+                        onSuccess: async () => {
+                            loading.hide();
+                            await queryClient.invalidateQueries({
+                                queryKey
+                            });
+                        },
+                        onError: (err: unknown) => {
+                            console.error(
```

Comment:
Shouldn't we hide loading when an error occurs?
Maybe also a `toast` with an error message so the user can notice it?